libs/native-federation-runtime/src/lib/init-federation.ts

```diff
@@ -18,13 +16,8 @@ export async function initFederation(
       ? await loadManifest(remotesOrManifestUrl)
       : remotesOrManifestUrl;
 
-  const hostImportMap = await processHostInfo();
   const remotesImportMap = await processRemoteInfos(remotes);
-
-  const importMap = mergeImportMaps(hostImportMap, remotesImportMap);
```

Comment:
#### Remote unnecessary global cache for loading the remotes
No DOM modifications.